packages/pn-commons/src/components/AppStatus/downtimeLog.utils.tsx

```diff
@@ -4,36 +4,38 @@ import DownloadIcon from '@mui/icons-material/Download';
 import { CardElement, Column, Item } from '../../types';
 import { DowntimeLogPage, DowntimeStatus } from '../../models';
 import { getLocalizedOrDefaultLabel } from '../../services/localization.service';
-import { formatDateTime } from '../../services/date.service';
-
+import { formatDateTime, formatDate, formatTime } from '../../utils/date.utility';
 export function booleanStringToBoolean(booleanString: string): boolean {
-  return booleanString.toLowerCase() === "true";
+  return booleanString.toLowerCase() === 'true';
 }
 
 /* eslint-disable-next-line arrow-body-style */
 const FormattedDateAndTime = ({ date, inTwoLines }: { date: string; inTwoLines?: boolean }) => {
   if (date) {
     const dateAndTime = formatDateTime(date);
-    return inTwoLines
-      ? <Stack direction="column">
-        <Typography variant="body2">{dateAndTime.date},</Typography>
-        <Typography variant="body2">{dateAndTime.time}</Typography>
+    return inTwoLines ? (
+      <Stack direction="column">
+        <Typography variant="body2">{formatDate(date)},</Typography>
+        <Typography variant="body2">{formatTime(date)}</Typography>
```

Comment:
A bug slipped here: the second line lacks the indication 'ore'
![image](https://user-images.githubusercontent.com/5519535/211319810-8e5cb7fe-709b-4d8b-9166-02aa107807f2.png)
which is present (correctly) in the mobile version
![image](https://user-images.githubusercontent.com/5519535/211320096-3378b2b1-bb4f-4ea7-9e42-2d804e980192.png)
I suggest adding a function `formatTimeWithLegend` which prepends the `date-time.hour-of-day` text to the time in HH:MM format.

Reply:
create function suggested